velox/exec/TableWriter.cpp

```diff
@@ -55,16 +53,17 @@ TableWriter::TableWriter(
   }
 
   mappedType_ = ROW(std::move(names), std::move(types));
-  const auto& config = driverCtx_->task->queryCtx()->config();
-  if (config.createEmptyFiles()) {
-    LOG(INFO) << "Enforce creating empty files\n";
-    createDataSink();
-  }
```

Comment:
[Update 10/31] Here I'm removing the logic of creating an empty file even when there is no input. This logic is only used by spark cpp now. Removing this helps simplify the logic and data structures.